exercises/rotational-cipher/Cargo.toml

```diff
@@ -0,0 +1,4 @@
+[package]
+name = "rotational-cipher"
+version = "0.0.0"
+authors = ["Chad Baxter <cbaxter@mail.umw.edu>"]
```

Comment:
note - the second exercise with an `authors` line (the first is nucleotide-codons).